docs/reference/hooks.rst

```diff
@@ -287,7 +287,8 @@ Hooks for building new areas of the admin interface (alongside pages, images, do
   :label: text displayed in the "Other Searches" option box.
   :name: an internal name used to identify the search option; defaults to the slugified form of the label.
   :url: the URL of the target search page.
-  :classnames: additional CSS classnames applied to the link, used to give it an icon.
+  :classnames: arbitrary CSS classnames applied to the link
+  :icon_name: icon to display next to the label.
```

Comment:
**Question (non-blocking):** do we have an overview of acceptable icon names? Maybe we could link to that?

Reply:
The only place I'm aware of that lists them is the Styleguide app that one can turn on in their Wagtail installation. Would be nice to have them on a page in the docs, too.